### DISCUSSION TITLE
Error when injecting found cross-seeds to qBittorrent using Torznab via Prowlarr

### DISCUSSION THREAD

**Discussion by kiwi3007** [cross-seed/cross-seed#215] [Q&A] [answered]
Cross-seed finds all torrents in my qBittorrent list and starts searching for them from tracker from Prowlarr
However, any it finds give:
`error: Unsupported: magnet link detected at [localhost]:[port]/10/download?apikey=[REDACTED]`

Pasting this link into my browser, and replacing redacted opens my qBittorrent WebUI, but doesn't result in an action beyond that.

**Comment by mmgoodnow** [comment, marked as answer]
Magnet links don't have enough data for cross-seed to match properly. You might be able to configure prowlarr to use torrent files instead of magnet links.
